packages/auto_mappr/analysis_options.yaml

```diff
@@ -5,6 +5,9 @@ analyzer:
   plugins:
     - dart_code_metrics
 
+  exclude:
+    - test/integration/fixture/error/**
```

Comment:
I think having `test/integration/error_fixture` would be better.

Reply:
Agree